pom.xml

```diff
@@ -18,7 +18,7 @@
 
     <properties>
         <dep.flink.version>1.5.0</dep.flink.version>
-        <dep.gradoop.version>0.4.0</dep.gradoop.version>
+        <dep.gradoop.version>0.5.0-SNAPSHOT</dep.gradoop.version>
```

Comment:
When Gradoop 0.5.0 is released, change to stable version.

<dep.gradoop.version>0.5.0</dep.gradoop.version>